### PR TITLE
Updating exception handing middleware sample to be more descriptive about how to return a specific status code in the http response

### PR DESCRIPTION
**Pull request by kshyju**
Updating exception handing middleware sample to be more descriptive about how to return a specific status code in the http response.

### CODE REVIEW

File: samples/CustomMiddleware/ExceptionHandlingMiddleware.cs [hunk 1]
```diff
@@ -36,8 +38,10 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
 
                 if (httpReqData != null)
                 {
-                    var newHttpResponse = httpReqData.CreateResponse();
-                    await newHttpResponse.WriteAsJsonAsync(new { FooStatus = "Invocation failed!" });
+                    // Create an instance of HttpResponseData with 500 status code.
+                    var newHttpResponse = httpReqData.CreateResponse(HttpStatusCode.InternalServerError);
+                    // You need to explicitly pass the status code in WriteAsJsonAsync method due to #776
```
Comment (madelinegordon) [inline samples/CustomMiddleware/ExceptionHandlingMiddleware.cs:43]: What does "#776" mean here?

Comment (madelinegordon) [reply samples/CustomMiddleware/ExceptionHandlingMiddleware.cs:43]: Ah nevermind I guess github just told me :)

Comment (fabiocav) [reply samples/CustomMiddleware/ExceptionHandlingMiddleware.cs:43]: Agreed. Let's have the full issue URL here.

Comment (kshyju) [reply samples/CustomMiddleware/ExceptionHandlingMiddleware.cs]: Switched to full issue link.

**Review by fabiocav — APPROVED** [review]

**Pull request merged**